dts/arm64/renesas/gen3_common/r8a7795_r8a7796_common.dtsi

```diff
@@ -0,0 +1,95 @@
+/*
+ * Device Tree Source for the R-Car H3/M3 (R8A7795/R8A7796) SoC
+ *
+ * Copyright (C) 2023 EPAM Systems.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <arm64/armv8-a.dtsi>
+#include <dt-bindings/interrupt-controller/arm-gic.h>
+#include <zephyr/dt-bindings/clock/renesas_cpg_mssr.h>
+#include <zephyr/dt-bindings/clock/r8a7795_cpg_mssr.h>
+
+/ {
+	#address-cells = <2>;
+	#size-cells = <2>;
+
+	cpus {
```

Comment:
It is not good idea to put `cpus` node into common DTSi. H3 and M3 has different set of CPUs.

Reply:
as far as know Renesas gen3 m3 and h3 boards use the same CPUs

Reply:
Yes and no. There are the same CPU models, but there are different count of CPUs. As I said, H3 and M3 has different **sets** of CPUs. Putting `cpus` node into common part will cause problems later, when we'll enable SMP and will be required to provide actual CPUs configuration.

Reply:
done